src/schema/v1/filter_artworks.ts

```diff
@@ -107,69 +111,40 @@ export const FilterArtworksType = new GraphQLObjectType<any, ResolverContext>({
         },
       }),
       resolve: (
-        { options: gravityOptions },
+        { options: gravityOptions, aggregations, hits },
```

Comment:
Now, this resolver, `artworks_connection`, doesn't ever fetch artworks (and needing to propagate all of the same filter options).

It just resolves based on the already fetched data for `filter_artworks`.

This also lets us get rid of some of this duplicated code that picks the correct loader based on options, etc.